migrate/github_to_json.py

```diff
@@ -312,11 +313,11 @@ def fetch_teams(gh_headers: dict, org: ApiOrganization) -> List[Team]:
     assert isinstance(admin_user_response.json(), list)
     admin_team = Team(
         slug=get_admin_team_slug(org.login),
-        members=list(
```

Comment:
members list didn't change at all; I just removed the spurious `list(...)` call and de-dented it.